compiler/qsc_lowerer/src/lib.rs

```diff
@@ -156,7 +156,19 @@ impl Lowerer {
     fn lower_item(&mut self, item: &hir::Item) -> fir::Item {
         let kind = match &item.kind {
             hir::ItemKind::Namespace(name, items) => {
-                let name = self.lower_ident(name);
+                let name = fir::Ident {
+                    // We don't have a node ID for the whole dotted name,
+                    // but we can use the node ID of the last part.
+                    // This ident is not a local anyway. (@swernli should check if this is harmless)
```

Comment:
@swernli for the check, afterwards that part of the comment can be removed.

Reply:
Yes, this looks safe, because it effectively matches what we were doing before (creating a mapping from an `Ident` to a `LocalVarId`), and it's true that this one should not get used as a real local variable anywhere because of how name resolution works for namespaces (and items, as it turns out).

Reply:
The comment can be removed.